cmake/OpenCVCompilerOptimizations.cmake

```diff
@@ -177,6 +179,7 @@ if(X86 OR X86_64)
     ocv_update(CPU_FMA3_FORCE "") # Don't force other optimizations
     ocv_update(CPU_FP16_IMPLIES "AVX")
     ocv_update(CPU_FP16_FORCE "") # Don't force other optimizations
+    ocv_update(CPU_XOP_IMPLIES "AVX")
```

Comment:
There is wiki page: https://en.wikipedia.org/wiki/XOP_instruction_set

`_mm_comge_epu32` can be emulated via SSE2 too. [Example](http://sseplus.sourceforge.net/group__emulated___s_s_e2.html#g17da283c41a855ae9c34c4782bbce9d9).

Reply:
There's no any AVX intrinsics involved only SSE. The reason of implies AVX its because all AMD CPUs that support XOP also support AVX and MSVC requires AVX too